src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -141,12 +153,37 @@ public Person toModelType() throws IllegalValueException {
         final Metadata modelMetadata = new Metadata(metadata);
 
         final Amount modelAmount = amount.toModelType();
+
+        if (attendance == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    Attendance.class.getSimpleName()));
+        }
+        if (!Attendance.isValidAttendance(attendance)) {
+            throw new IllegalValueException(Attendance.MESSAGE_CONSTRAINTS);
+        }
+        final Attendance modelAtt = new Attendance(attendance);
+
+        if (sessions == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    Sessions.class.getSimpleName()));
+        }
+        if (!Sessions.isValidSessions(sessions)) {
+            throw new IllegalValueException(Sessions.MESSAGE_CONSTRAINTS);
+        }
+        final Sessions modelSess = new Sessions(sessions);
+
+        if (isMoreThanSess(modelAtt, modelSess)) {
+            throw new IllegalValueException(Attendance.MESSAGE_LESS_THAN_CONSTRAINT);
+        }
         final Address modelAddress = new Address(address);
         final Set<Role> modelRoles = new HashSet<>(personRoles);
         final Set<Cca> modelCcas = new HashSet<>(personCcas);
 
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelRoles,
-                modelCcas, modelAmount, modelMetadata);
+        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelRoles, modelCcas, modelAmount,
+                modelAtt, modelSess, , modelMetadata);
```

Comment:
There is a empty parameter - you have to fix this error

Reply:
Thank you. Fixed